jena-rdfconnection/src/main/java/org/apache/jena/rdfconnection/examples/RDFConnectionExample6.java

```diff
@@ -40,4 +40,3 @@ public static void main(String ...args) {
         }
     }
 }
-
```

Comment:
I thought git asked for this extra newline at the end of the file.

Reply:
Git copes either way (I think). These example changes are removing an extra blank line. It was "}\n\n".